gradle/dependencies.gradle

```diff
@@ -73,7 +73,11 @@ versions += [
   zkclient: "0.10",
   zookeeper: "3.4.10",
   jfreechart: "1.0.0",
-  mavenArtifact: "3.5.2"
+  mavenArtifact: "3.5.2",
+  avro4sVersion: "1.8.3",
+  scalaLoggingVersion: "3.8.0",
```

Comment:
`scalaLogging` is already defined in this file.

Reply:
removed.